lib/wor/paginate/formatter.rb

```diff
@@ -32,7 +32,9 @@ def paginated_content
       end
 
       def serialized_content
-        return paginated_content.map { |item| serializer.new(item) } if serializer.present?
+        if serializer.present?
```

Comment:
i would rather try to use it in one line, we try to avoid the `if - end`
```
return paginated_content.map { |item| serializer.new(item, @options) } if serializer.present?
```
if rubocop says it's too long you could break it after the if and before the condition
```
return paginated_content.map { |item| serializer.new(item, @options) } if 
  serializer.present?
```

Reply:
Another option could by use a lambda
```
new_serializer = ->(item) { serializer.new(item, @options) }
return paginated_content.map(&new_serializer) if serializer.present?
```